src/util.ts

```diff
@@ -109,38 +157,61 @@ export function providersFiles(
   return files;
 }
 
-export async function localProviders(
-  superJson = loadSuperJson()
-): Promise<string[]> {
-  const cwd = await SuperJson.detectSuperJson(process.cwd());
+export async function localProviders(options?: {
+  fileSystem?: IFileSystem;
+}): Promise<string[]> {
+  const cwd = await detectSuperJson(
+    process.cwd(),
+    options?.fileSystem ?? NodeFileSystem
+  );
+
+  if (cwd === undefined) {
+    throw new Error('Super.json not found');
+  }
 
-  return glob
-    .sync('../providers/*.json', { cwd })
-    .map(i => normalizePath(i, superJson));
+  return Promise.all(
+    glob
+      .sync('../providers/*.json', { cwd })
+      .map(async i => normalizePath(cwd, i))
+  );
```

Comment:
suggestion: Shouldn't this be only:

```suggestion
  return glob
    .sync('../providers/*.json', { cwd })
    .map(i => normalizePath(cwd, i));
```

Reply:
Yes, thanks.